Superformula/flutter_test-2023
# Streamline Requirements for Coding Challenge.

_**Goal:**_  We want to want to streamline the code challenge while focusing on the pieces that matter.

**_Explanation_**

- Simplifying the Figma file by removing atomic/molecule, fonts, sizing, and paddings from the specification.
- Calling out that time does not need to be spent on the pixel perfection of implementation. It's always the last five percent of work that takes the most time.
- Removing any bonus goals. We do not want submitters to feel obligated to implement bonus items. They are not longer bonus items if that is the case.
- Specifying that the unit test should be of their state management solution.
- Removing the unnecessary complexity of the "Load More" button.

**_Validation_**

- Pull down and run the project.
- Verify that there are no linter errors and the code successfully runs.
- Press the button and verify that it returns `flutter: Fetched 0 restaurants`.
- DM Matt for an API key and place inside of `yelp_repository.dart`.
- Press the button and verify that it returns `flutter: Fetched 20 restaurants`.
- View the updated MarkDown and verify the new text/images render correctly.
- Follow the Figma linked and verify it only contains a ListView and DetailView.
